# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cdrtools/linux-2.6.7-__user.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./libscg/scsi-linux-sg.c.orig	2002-10-21 19:20:35.000000000 +0300
+++ ./libscg/scsi-linux-sg.c	2004-06-27 15:23:02.670972688 +0300
@@ -78,6 +78,10 @@
 #include "scsi/scsi.h"
 #endif
 
+#if LINUX_VERSION_CODE >= 0x020607
+#include <linux/compiler.h>
+#endif
+
 #include "scsi/sg.h"
 
 #undef sense			/* conflict in struct cdrom_generic_command */
